app/views/record/_record.html.erb

```diff
@@ -1,59 +1,183 @@
+    <h3 class="section-title">Links</h3>
+    <ul class="list-links">
+      <li>
+        <span class="label">Source:</span> <%= link_to(@record['source'], @record['sourceLink']) %>
+      </li>
+
+      <% if @record['links'].present? %>
+        <% @record['links'].each do |link| %>
+          <li>
+            <%= link_to(link['text'] || 'unknown', link['url']) %>
+          </li>
+        <% end %>
+      <% end %>
+    </ul>
+
+    <% if @record['summary'].present? %>
+      <h3 class="section-title">Summary</h3>
+      <% @record['summary'].each do |paragraph| %>
+        <p>
+          <%= sanitize paragraph, tags: %w(p strong em a), attributes: %w(href) %>
+        </p>
+      <% end %>
+    <% end %>
+
+    <h3 class="section-title">More information</h3>
+
+    <ul class="list-moreinfo">
+      <%# pub type%>
+
+      <%# pub info%>
+      <% if @record['publicationInformation'] %>
+        <li>Publication Information: <%= @record['publicationInformation'].join('; ') %></li>
+      <% end %>
+
+      <%# identifiers %>
+      <% if @record['identifiers'].present? %>
+
+        <% @record['identifiers'].each do |id| %>
+          <li>
+            <span class="label"><%= id['kind'].upcase %>:</span> 
+            <span class="<%= id['kind']%>"><%= id['value'] %></span>
+          </li>
+        <% end%>
+      <% end %>
+
+      <%# language %>
+      <% if @record['languages'].present? %>
+        <li>
+          <span class="label"><%= "Language".pluralize(@record['languages'].count) %>:</span> <%= @record['languages'].join(',') %>
+        </li>
+      <% end %>
+    </ul>
+
+    <%# subjects%>
+    <% if @record['subjects'].present? %>
+      <h3 class="section-title">Subjects</h3>
+        <ul class="list-subjects">
+          <% @record['subjects'].each do |subject| %>
+            <li>
+              <%= "#{subject['kind']}: " if subject['kind'] != "Subject scheme not provided" %>
+              <%= subject['value'].join(', ') %>
+            </li>
+          <% end %>
+        </ul>
+    <% end %>
+
+    <%# dates%>
+    <% if @record['dates'].present? %>
+      <h3 class="section-title">Dates</h3>
+        <ul class="list-dates">
+          <% @record['dates'].each do |date| %>
+            <li>
+              <%= date['kind'] %>: <%= date_parse(date['value']) %><%= date_range(date['range']) %>
+              <%= " Note: #{date['note']}" if date['note'].present? %>
+            </li>
+          <% end %>
+        </ul>
+    <% end %>
+
+    <% if @record['notes'].present? %>
+      <h3 class="section-title">Notes</h3>
+      <% @record['notes'].each do |note| %>
+        <% if note['kind'].present? %>
+          <%= note['kind'] %>:
+        <% end %>
+        <% note['value'].each do |paragraph| %>
+          <%= sanitize paragraph, tags: %w(p strong em a), attributes: %w(href) %>
+        <% end %>
+        </p>
```

Comment:
I think this is an extraneous tag?

Reply:
Yeah I have no idea where that came from. Thanks for catching it!